source/manual/alerts/app-healthcheck-failed.html.md

```diff
@@ -15,3 +15,9 @@ endpoint doesn't respond successfully, it triggers this alert.
 
 To solve this issue, look at the logs of the application to see what
 is wrong.
+
+## Application healthcheck documentation
```

Comment:
There are other apps with healthchecks in the dev docs.  I think they should all be in this list, or it shouldn't be here.